src/experiments/kfold_opportunity.py

```diff
@@ -0,0 +1,96 @@
+"""
+test with new config
+
```

Comment:
I like the idea of a description of the experiment at the top of the file, but this is not a helpful description :D

- What is trained?
- How is it evaluated?
- (E.g. how long does it take)

```suggestion
K-Fold (Leave recording out) evaluation of JensModel saving accuracy, confusion matrix and subject information to an experiment folder

This uses the new DataConfig.

Expected accuracy: ~xx%
Takes around 6 minutes on DHC lab
```